### PR TITLE
rename CLI to `ta`

### LINKED ISSUE #222: Rename cli to `ta`

### PR DESCRIPTION
**Pull request by djmitche**
As decided in #157.  Fixes #222.

### CODE REVIEW

**Comment by dbr** [comment]
One question above but aside from that looks good to me!

**Comment by djmitche** [comment]
Is that question still in an un-saved buffer somewhere?

**Comment by dbr** [comment]
> Is that question still in an un-saved buffer somewhere?

Woops, yes - I just realized the line comments start a "Review" thing which requires a subsequent button to be clicked!

File: cli/src/usage.rs [hunk 1]
```diff
@@ -70,10 +70,10 @@ impl Usage {
             )
         )?;
         for modification in self.modifications.iter() {
-            modification.write_help(&mut w, summary)?;
+            modification.write_help(&mut w, command_name, summary)?;
         }
         if !summary {
-            writeln!(w, "\nSee `task help` for more detail")?;
+            writeln!(w, "\nSee `ta help` for more detail")?;
```
[the remark this answers is not in the document]
Comment (djmitche) [reply cli/src/usage.rs:76]: Ah, yes, probably!

**Comment by djmitche** [comment]
Fixed and rebased.  I'll merge if it's green.

**Pull request merged**